src/pages/Home.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Home = () => (
+  <div>
+    <h1>Home</h1>
+    <p>Book Store</p>
+  </div>
+);
```

Comment:
This component which serves as both the Home and Books component is required to render a static list of books.

Kindly create and map over an array to render them to the page.

<img width="594" alt="image" src="https://github.com/Othniel2471/book-store/assets/27709832/84cde3a3-88c9-45ed-ae4e-f15d01f1de4e">

Reply:
@brytebee  please is it going to be dynamic, or can I just create an object of books and then render it, or am I meant to write the whole functionality for adding the books dynamically?

Reply:
@Othniel2471 It can be a static book such as BookList. Then just map over it to render them on the page.